test/integration/user_login_test.rb

```diff
@@ -14,16 +14,13 @@ def setup
   end
 
   test "user can login" do
-    within ("#header") do
-      assert page.has_content?('Coffee House')
-    end
+    assert page.has_content?('Coffee House')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.